### ISSUE TITLE
Menu items cannot be selected by VoiceOver

### ISSUE THREAD

**Issue by devongovett** [tailwindlabs/headlessui#62]
When testing the [menu example](https://codesandbox.io/s/headlessuireact-menu-example-b6xje?from-embed) on iOS, you can focus an item in the menu but double tapping with VoiceOver to select the item has no effect. The same can be seen on macOS if you use the Ctrl + Option + Space command that usually triggers a click on an item. I see it change the hash in the url but the menu does not close.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

This was indeed a bug in a previous version (0.1.3), but has been fixed in a newer version. A version bump in our demo example will fix it. I will ping Adam to update it in his example!

Thanks!
